tools/content-validation/eng/pipelines/python/run-all-packages.yml

```diff
@@ -0,0 +1,88 @@
+parameters:
+- name: packages
+  type: object
+  displayName: |
+    A list of package names to validate.
+  default:
+    - azure-ai-contentsafety
```

Comment:
While reducing the packages to just a list here is a step in the right direction I think we can still do better. Can we figure out a way to dynamically find the list of packages so that we don't have to maintain this and we don't have to worry about missing any?

Reply:
One way we can think of at the moment is to access this [page](https://learn.microsoft.com/en-us/python/api/overview/azure/?view=azure-python-preview) through playwright and get all the packages according to the service. However, there is a problem, not all packages in our test plan need to be tested, such as management-related packages, abandoned packages, and some packages that do not need to be tested. We need to filter these out. This means that even if we automatically get a list of packages that need to be tested, we need to configure the list of packages that need to be filtered. cc @xiangyan99 Any ideas?

Reply:
What are the criteria for filtering out? Can it be checked in code without needing a hard-coded list?

Reply:
Due to changes in the test scope (the scope of “all” now refers to the packages released last month), we’ve updated the configuration here. It now uses dynamic fetching, and the hard-coded list has been removed.